src/main/java/com/course/fleet/management/api/controller/UserController.java

```diff
@@ -0,0 +1,72 @@
+package com.course.fleet.management.api.controller;
+
+import com.course.fleet.management.api.entity.User;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import net.datafaker.Faker;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/users")
```

Comment:
Recomiendo mantener el recurso en singular. Esto ya que usualmente las operaciones base son crear, actualizar, obtener y eliminar 1 solo usuario.

Una recomendación es que al endpoint de obtener listado puedes agregarle el path "/list" o "/all"
```suggestion
@RequestMapping("/user")
```